homeassistant/components/ebusd/const.py

```diff
@@ -0,0 +1,84 @@
+"""Constants for ebus component."""
+DOMAIN = 'ebusd'
+
+READ_COMMAND = 'read -m {2} -c {0} {1}\n'
```

Comment:
All device specific interface code should be moved to the library.

Reply:
Moved to library